### PR TITLE
More Coffee Mug Flavor Text

### PR DESCRIPTION
**Pull request by DanteTorn**
#### Summary

SUMMARY: Content "Added more coffee mug flavor text and cleaned up some of the formatting of what was already there."

#### Purpose of change

More fun mug descriptions to find and collect.
Formatting changes make it the code more readable.

#### Additional context

I had a bunch more of these listed down but pared down the list to what is in this PR. Most of them weren't very good.

### CODE REVIEW

**Comment by Fris0uman** [comment]
You need to lin your changes https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md

**Comment by DanteTorn** [comment]
I've ran it through the linter and it's showing as replaced the whole thing. I've spoken to esotericist and it seems that my thing is configured to use windows line endings instead of unix. I'm going to configure whatever the proper term for the thing is to fix this so this issue for this PR so doesn't come up again in future PRs I make.

File: data/json/items/generic/dining_kitchen.json [hunk 1]
```diff
@@ -187,7 +187,35 @@
         "text": "The side of the mug reads 'Do you even linux, bro?' and has a picture of a muscular penguin."
       },
       { "id": "mug15", "text": "The side of the mug reads 'I wish this was wine!'" },
-      { "id": "mug16", "text": "The side of the mug reads 'CasUaL aLcoHoLiSm'" }
+      { "id": "mug16", "text": "The side of the mug reads 'CasUaL aLcoHoLiSm'" },
+      { "id": "mug17", "text": "A white ceramic coffee mug. It says \"World's Best Dad!\" on the side." },
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:191]:
```suggestion
      { "id": "mug17", "text": "A white ceramic coffee mug.  It says \"World's Best Dad!\" on the side." },
```

File: data/json/items/generic/dining_kitchen.json [hunk 2]
```diff
@@ -187,7 +187,35 @@
...
+      {
+        "id": "mug18",
+        "text": "A white ceramic coffee mug. It says \"Great Ideas start with Great Coffee.\" on the side."
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:194]:
```suggestion
        "text": "A white ceramic coffee mug.  It says \"Great Ideas start with Great Coffee.\" on the side."
```

File: data/json/items/generic/dining_kitchen.json [hunk 3]
```diff
@@ -187,7 +187,35 @@
...
+      },
+      {
+        "id": "mug19",
+        "text": "A white ceramic coffee mug. On the side is a glass of wine and a cup of coffee both saying \"She loves me more!\""
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:198]:
```suggestion
        "text": "A white ceramic coffee mug.  On the side is a glass of wine and a cup of coffee both saying \"She loves me more!\""
```

File: data/json/items/generic/dining_kitchen.json [hunk 4]
```diff
@@ -187,7 +187,35 @@
...
+      },
+      {
+        "id": "mug20",
+        "text": "A white ceramic coffee mug. It says \"This is a designer mug. It fuels designers.\" on the side."
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:202]:
```suggestion
        "text": "A white ceramic coffee mug.  It says \"This is a designer mug. It fuels designers.\" on the side."
```

File: data/json/items/generic/dining_kitchen.json [hunk 5]
```diff
@@ -187,7 +187,35 @@
...
+      },
+      { "id": "mug21", "text": "A white ceramic coffee mug. On the side is a winking woman in lingerie." },
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:204]:
```suggestion
      { "id": "mug21", "text": "A white ceramic coffee mug.  On the side is a winking woman in lingerie." },
```

File: data/json/items/generic/dining_kitchen.json [hunk 6]
```diff
@@ -187,7 +187,35 @@
...
+      {
+        "id": "mug22",
+        "text": "A white ceramic coffee mug. It says \"You're my definition of perfect. <3\" on the side."
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:207]:
```suggestion
        "text": "A white ceramic coffee mug.  It says \"You're my definition of perfect. <3\" on the side."
```

File: data/json/items/generic/dining_kitchen.json [hunk 7]
```diff
@@ -187,7 +187,35 @@
...
+      },
+      {
+        "id": "mug23",
+        "text": "A white ceramic coffee mug. It says \"You've been poisoned.\" on the bottom, inside of it."
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:211]:
```suggestion
        "text": "A white ceramic coffee mug.  It says \"You've been poisoned.\" on the bottom, inside of it."
```

File: data/json/items/generic/dining_kitchen.json [hunk 8]
```diff
@@ -187,7 +187,35 @@
...
+      },
+      {
+        "id": "mug24",
+        "text": "A white ceramic coffee mug. On the side is a dog wearing a hat, sat in a burning room, saying \"This is fine.\""
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:215]:
```suggestion
        "text": "A white ceramic coffee mug.  On the side is a dog wearing a hat, sat in a burning room, saying \"This is fine.\""
```

File: data/json/items/generic/dining_kitchen.json [hunk 9]
```diff
@@ -187,7 +187,35 @@
...
+      },
+      { "id": "mug25", "text": "A white ceramic coffee mug. There's a tiny dinosaur sculpture inside of it." },
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:217]:
```suggestion
      { "id": "mug25", "text": "A white ceramic coffee mug.  There's a tiny dinosaur sculpture inside of it." },
```

File: data/json/items/generic/dining_kitchen.json [hunk 10]
```diff
@@ -187,7 +187,35 @@
...
+      { "id": "mug26", "text": "A white ceramic coffee mug. It says \"I <3 Massachusetts.\" on the side." }
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:218]:
```suggestion
      { "id": "mug26", "text": "A white ceramic coffee mug.  It says \"I <3 Massachusetts.\" on the side." }
```

**Comment by ZhilkinSerg** [comment]
See https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md

File: data/json/items/generic/dining_kitchen.json [hunk 11]
```diff
@@ -230,7 +258,7 @@
     "id": "tin_cup",
     "name": { "str": "tin cup" },
     "symbol": "u",
-    "description": "An enameled tin cup.  Great for camping or for prison use; makes a wonderful sound when clanged along bars.",
+    "description": "An enameled tin cup. Great for camping or for prison use; makes a wonderful sound when clanged along bars.",
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:261]:
```suggestion
    "description": "An enameled tin cup.  Great for camping or for prison use; makes a wonderful sound when clanged along bars.",
```

File: data/json/items/generic/dining_kitchen.json [hunk 12]
```diff
@@ -210,7 +210,35 @@
...
+        "text": "A white ceramic coffee mug.  On the side is a glass of wine and a cup of coffee both saying \"She loves me more!\""
+      },
+      {
+        "id": "mug20",
+        "text": "A white ceramic coffee mug.  It says \"This is a designer mug. It fuels designers.\" on the side."
+      },
+      { "id": "mug21", "text": "A white ceramic coffee mug.  On the side is a winking woman in lingerie." },
+      {
+        "id": "mug22",
+        "text": "A white ceramic coffee mug.  It says \"You're my definition of perfect. <3\" on the side."
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:230]:
```suggestion
        "text": "A white ceramic coffee mug.  It says \"You're my definition of perfect.  <3\" on the side."
```

File: data/json/items/generic/dining_kitchen.json [hunk 13]
```diff
@@ -210,7 +210,35 @@
...
+        "id": "mug18",
+        "text": "A white ceramic coffee mug.  It says \"Great Ideas start with Great Coffee.\" on the side."
+      },
+      {
+        "id": "mug19",
+        "text": "A white ceramic coffee mug.  On the side is a glass of wine and a cup of coffee both saying \"She loves me more!\""
+      },
+      {
+        "id": "mug20",
+        "text": "A white ceramic coffee mug.  It says \"This is a designer mug. It fuels designers.\" on the side."
```
Comment (ZhilkinSerg) [inline data/json/items/generic/dining_kitchen.json:225]:
```suggestion
        "text": "A white ceramic coffee mug.  It says \"This is a designer mug.  It fuels designers.\" on the side."
```

**Pull request merged**